examples/fastlib/resources/base.py

```
@@ -0,0 +1,31 @@
+from typing import NoReturn
```

Ruff (0.8.2)

`typing.NoReturn` imported but unused

Remove unused import: `typing.NoReturn`

(F401)